app/src/main/java/com/example/concordia_campus_guide/Models/Building.java

```diff
@@ -1,27 +1,39 @@
 package com.example.concordia_campus_guide.Models;
 
+
 import android.text.TextUtils;
 
+import com.google.android.gms.maps.model.GroundOverlayOptions;
+import com.google.android.gms.maps.model.LatLng;
+
 import java.util.List;
 
 public class Building {
 
+    private LatLng centerCoordinates;
+    private String[] availableFloors;
+    private String buildingCode;
+    private float width;
+    private float height;
+    private float bearing;
     private String Campus;
     private String Building_Code;
     private String Building_Long_Name;
     private String Address;
     private List<String> Departments;
     private List<String> Services;
 
-    /**
-     * Constructor
-     */
-    public Building(){
-    }
+    private GroundOverlayOptions groundOverlayOption;
 
-    /**
-     * Constructor
-     */
+    public Building(LatLng centerCoordinates, String[] availableFloors, String buildingCode, float width, float height, float bearing) {
```

Comment:
So, you and Thomas created the same class ?

Reply:
yes, I will fix conflicts